staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/converter.go

```diff
@@ -173,6 +173,13 @@ func (c *delegatingCRConverter) ConvertToVersion(in runtime.Object, target runti
 		// TODO: should this be a typed error?
 		return nil, fmt.Errorf("%v is unstructured and is not suitable for converting to %q", fromGVK.String(), target)
 	}
+
+	// A smoke test in API machinery calls the converter on empty objects during startup. The test is initiated here:
+	// https://github.com/kubernetes/kubernetes/blob/dbb448bbdcb9e440eee57024ffa5f1698956a054/staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go#L201
+	if isEmptyUnstructuredObject(in) {
+		return NewNOPConverter().Convert(in, toGVK.GroupVersion())
+	}
+
```

Comment:
this should happen after all the scale special cases and validversions checks, just before delegating to `c.converter.Convert`

Reply:
Fixed